gensim/models/doc2vec.py

```diff
@@ -59,16 +60,29 @@
 
 try:
     from gensim.models.doc2vec_inner import train_document_dbow, train_document_dm, train_document_dm_concat
-    from gensim.models.word2vec_inner import FAST_VERSION  # blas-adaptation shared from word2vec
+    # blas-adaptation shared from word2vec
+    from gensim.models.word2vec_inner import FAST_VERSION
     logger.debug('Fast version of {0} is being used'.format(__name__))
 except ImportError:
     logger.warning('Slow version of {0} is being used'.format(__name__))
-    # failed... fall back to plain numpy (20-80x slower training than the above)
+    # failed... fall back to plain numpy (20-80x slower training than the
+    # above)
     FAST_VERSION = -1
 
-    def train_document_dbow(model, doc_words, doctag_indexes, alpha, work=None,
-                            train_words=False, learn_doctags=True, learn_words=True, learn_hidden=True,
-                            word_vectors=None, word_locks=None, doctag_vectors=None, doctag_locks=None):
+    def train_document_dbow(
+            model,
+            doc_words,
+            doctag_indexes,
+            alpha,
+            work=None,
+            train_words=False,
+            learn_doctags=True,
+            learn_words=True,
+            learn_hidden=True,
+            word_vectors=None,
+            word_locks=None,
+            doctag_vectors=None,
+            doctag_locks=None):
```

Comment:
Is this explosion of parameters, one-to-a-line, really in PEP8? (I didn't notice it on a quick review, and I find it makes the code less readable by taking up so much vertical-space/scrolling between function-begin and function-logic.)